dev_guide/deployments.adoc

```diff
@@ -110,28 +110,61 @@ created each time a new version of the `origin-ruby-sample:latest` image reposit
 ====
 
 == Deployment Client Tool Operations
```

Comment:
To better match other Dev Guide topics, how about either re-titling this "Using the CLI" (a la [1]), or maybe better yet just dropping the heading & intro sentence and let them be their own `==`-level headings (a la [2][3]).

[1] http://docs.openshift.org/latest/dev_guide/new_app.html
[2] http://docs.openshift.org/latest/dev_guide/projects.html
[3] http://docs.openshift.org/latest/dev_guide/builds.html